Description: Add 'logitheque' keyword
Author: Scibian Packaging Team <packaging@scibian.org>

--- a/data/gpk-application.desktop.in
+++ b/data/gpk-application.desktop.in
@@ -6,7 +6,7 @@
 Terminal=false
 Type=Application
 Categories=GNOME;GTK;System;Settings;PackageManager;
-_Keywords=Updates;Upgrade;Sources;Repositories;Preferences;Install;Store;
+_Keywords=Updates;Upgrade;Sources;Repositories;Preferences;Install;Store;Logitheque;
 NotShowIn=KDE;
 StartupNotify=true
 X-GNOME-UsesNotifications=true
